include/water.h

```diff
@@ -0,0 +1,6 @@
+#ifndef GUARD_WATER_H
+#define GUARD_WATER_H
+
+void sub_8108C94(struct Sprite *sprite);
```

Comment:
This should go in battle_anim.  `water.c` is just some battle anim code for water effects.